packages/main/src/themes/sap_belize_hcw/sizes-parameters.css

```diff
@@ -0,0 +1,17 @@
+@import "../base/sizes-parameters.css";
+
+:root {
+	/* SplitButton */
+	--_ui5_split_button_middle_separator_top: 0;
+	--_ui5_split_button_middle_separator_height: 2.5rem;
+	--_ui5_split_button_text_button_width: 2.5rem;
+}
+
+[data-ui5-compact-size],
+.ui5-content-density-compact,
+.sapUiSizeCompact {
+	/* SplitButton */
+	--_ui5_split_button_middle_separator_top: 0;
```

Comment:
also here

Reply:
ToggleButton and SegmentedButton (HCB & HCW) - in tog hover & down is not correct color

Reply:
Done

Reply:
Transparent SplitButton in HCB have wrong background color (disabled state)